tutorials/3d/baked_lightmaps.rst

```diff
@@ -157,65 +173,173 @@ can be controlled from the **Bake Mode** menu in lights:
 ------
 
-To begin the bake process, just push the big **Bake Lightmaps** button on top
-when selecting the ``BakedLightmap`` node:
+To begin the bake process, just push the **Bake Lightmaps** button on top
+when selecting the BakedLightmap node:
 
 .. image:: img/baked_light_bake.png
 
 This can take from seconds to minutes (or hours) depending on scene size, bake
 method and quality selected.
 
+Balancing bake times with quality
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since high-quality bakes can take very long (up to several hours for large complex scenes),
+it is recommended to use lower quality settings at first. Then, once you are confident
+with your scene's lighting setup, raise the quality settings and perform a "final"
+bake before exporting your project.
+
 Configuring bake
 ~~~~~~~~~~~~~~~~
 
 Several more options are present for baking:
 
-- **Bake Cell Size**: The Godot lightmapper uses a grid to transfer light information around; the default value is fine and should work for most cases. Increase it in case you want better lighting on small details or your scene is large.
-- **Capture Cell Size**: This is the grid used for real-time capture information (lighting dynamic objects). Default value is generally OK, it's usually smaller than Bake Cell Size and can't be larger than it.
-- **Bake Quality**: Three bake quality modes are provided, Low, Medium and High. Higher quality takes more time.
-- **Bake Mode**: The baker can use two different techniques: *ConeTract* (fast, but approximate), or *RayTrace* (slow, but accurate). If you run into issues, switch the bake mode to *RayTrace*.
-- **Bake Propagation**: Used for the *Voxel Cone Trace* mode. Works just like in :ref:`GIProbe <doc_gi_probes>`.
-- **Bake Hdr**: If disabled, lightmaps are smaller, but can't capture any light over white (1.0).
-- **Bake Extents**: Size of the area affected (can be edited visually)
-- **Image Path**: Where lightmaps will be saved. By default, on the same directory as the scene ("."), but can be tweaked.
-- **Light Data**: Contains the light baked data after baking. Textures are saved to disk, but this also contains the capture data for dynamic objects, which can be heavy. If you are using a scene in ``.tscn`` format, you should save this resource to an external binary ``.res`` file to avoid bloating the ``.tscn`` scene with binary data encoded in Base64.
+- **Bake Extents**: The size of the area affected. This can be edited in the 3D
+  editor viewport using the handles. Any object that can have lightmaps baked and
+  is *touching* the bake extents will have lightmaps baked for it, but dynamic
+  object capture will only work within the extents.
+
+Tweaks
+^^^^^^
+
+- **Quality:** Three bake quality modes are provided: Low, Medium, High, and Ultra.
+  Higher quality takes more time, but result in a better-looking lightmap with
+  less noise. The difference is especially noticeable with emissive materials or
+  areas that get little to no direct lighting.
```

Comment:
We can mention here that the number of samples for each quality level can be customized in the project settings at `rendering/cpu_lightmapper/quality/*_quality_ray_count`